test/TorchSharpTest/TestTraining.cs

```diff
@@ -16,7 +16,6 @@ namespace TorchSharp
 {
     public class TestTraining
     {
-
```

Comment:
Superfluous reformatting change.

Reply:
Damn, sorry for all the work this caused, I'll try to keep rider on a tighter leash from now on!

Reply:
Since I assume there are going to be massive changes in the next revision of this PR, it may be cleaner to close this one and then open up a new one. Not sure.